workers/unity/Packages/io.improbable.gdk.debug/WorkerInspector/EntityDetail.cs

```diff
@@ -29,6 +29,11 @@ public EntityDetail()
             entityName = this.Q<Label>("entity-name");
             entityId = this.Q<Label>("entity-id");
             componentContainer = this.Q<ScrollView>();
+            var window = EditorWindow.GetWindow<WorkerInspectorWindow>();
+            if (window != null)
+            {
+                window.OnToggleHideCollections += ComponentVisualElement.ToggleHideIfEmpty;
+            }
```

Comment:
This doesn't feel like the correct place for this callback to be registered. Was there a particular reason this happens here?

Reply:
I wanted to avoid introducing the dependency on `ComponentVisualElement` in `WorkerInspectorWindow` because it wasn't there before. But if you feel it isn't a problem, I'll move it "up".